internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -132,15 +132,20 @@ func (ksd KubernetesServiceDeployer) installCustomDefinitions() error {
 
 var _ ServiceDeployer = new(KubernetesServiceDeployer)
 
-func findKubernetesDefinitions(definitionsDir string) ([]string, error) {
-	files, err := filepath.Glob(filepath.Join(definitionsDir, "*.yaml"))
-	if err != nil {
-		return nil, errors.Wrapf(err, "can't read definitions directory (path: %s)", definitionsDir)
+func containsKustomization(definitionsPath string) (bool, error) {
+	if _, err := os.Stat(definitionsPath); err != nil {
+		return false, errors.Wrapf(err, "can't read definitions directory (path: %s)", definitionsPath)
 	}
-
-	var definitionPaths []string
-	definitionPaths = append(definitionPaths, files...)
-	return definitionPaths, nil
+	// check if kustomization.yaml exists
+	if _, err := os.Stat(filepath.Join(definitionsPath, "kustomization.yaml")); err != nil {
+		// if it does not exist, then the .empty file needs to be present
+		if _, err := os.Stat(filepath.Join(definitionsPath, ".empty")); err != nil {
+			return false, errors.Errorf("kustomization.yaml file is missing (path: %s). Add one or create an .empty file"+
+				" if no custom definitions are required.", definitionsPath)
```

Comment:
If we require certain files to be present, it would be better to make these checks on the package-spec.